packages/editor/src/components/url-input/index.js

```diff
@@ -242,27 +241,4 @@ class URLInput extends Component {
 	}
 }
 
-// TODO: As part of deprecation of UrlInput, the temporary passthrough
```

Comment:
Thanks to whomever wrote these deprecation comments, because they make reviewing this code removal very straightforward 👍